rustdesk-server: remove shell wrapper for init

AutoCONFIG opened the pull request:
rustdesk-server: 去除init脚本中的hbbr和hbbs启动命令的shell包装
1.硬编码日志路径到/var/run/rustdesk-server
2.硬编码密钥生成路径到/etc/rustdesk-server
3.硬编码数据库文件到/var/run/rustdesk-server
4.硬编码配置文件到/var/run/rustdesk-server
5.增加无法获取mac地址的退回机制

其中：
1. 删除HOME环境变量，使用硬编码机制将配置文件生成到/var/run/rustdesk-server。源代码中所使用的库会首先获取环境变量的HOME目录，无法获取则退回到passwd中的用户路径，而openwrt中系统用户的路径一般为/var/run/系统用户名，则将运行路径硬编码为/var/run/rustdesk-server，相对符合习惯。
2. /var/run/rustdesk-server路径下的内容重启后会丢失，因其数据库文件、配置文件是运行中实时生成的，且会积累，需要定期清除，故硬编码到此路径可以减少对闪存的不必要的写入/擦除磨损。
3. 虽然源代码支持直接的日志输出，但是日志往往很多很杂，如果在init中启用stdout会使得系统日志可阅读性降低，故改为硬编码到本地的运行路径/var/run/rustdesk-server，这样即方便获取调试日志也不太影响系统日志的可阅读性。
4. 虽然几乎不会发生，但依然考虑到潜在的无法获取mac地址的情况，导致无法生成id，所以增加此情况下的随机id生成，增加程序运行的支持度。
5. 最佳解决方案依然是openwrt的procd服务支持设定工作路径的功能，本次提交patch只是在增加此功能前的兼容方案。

已经过约一周的测试，运行稳定。测试环境：r66s（rk3568 arm64）,openwrt 24.10(tag代码编译)

1715173329 commented on net/rustdesk-server/files/rustdesk-server.init:
```diff
@@ -21,17 +22,20 @@ start_service() {
 		procd_set_param respawn
-		procd_set_param env HOME="$WORKDIR"
 		procd_set_param user rustdesk-server
 		procd_set_param group rustdesk-server
```
开 jail 看看能不能工作
procd_add_jail hbbr log procfs sysfs
procd_add_jail_mount "/etc/ssl/"
procd_add_jail_mount_rw "/etc/rustdesk-server/"
procd_add_jail_mount_rw "/var/run/rustdesk-server/"

1715173329 commented on net/rustdesk-server/patches/001-hardcode-log-path.patch:
```diff
@@ -0,0 +1,46 @@
+-        .log_to_stdout()
++        .log_to_file(
++            FileSpec::default()
++                .directory("/var/run/rustdesk-server")
```
建议添加 stop_service 以对日志文件进行回收（这里的话不用加 reload_service 了）。

AutoCONFIG replied:
目前这个代码会在每次启动时自动清除掉上次运行保存的log（即不是追加模式，是重写模式），虽然不会在stop_service时清除，但是也能达到差不多的效果，感觉可以不需要额外的回收机制

AutoCONFIG replied:
请教一下，openwrt似乎不使用/etc/default路径来放置程序要用到的环境变量，考虑到rustdesk-server的一些参数是由环境变量传递，而非命令参数，所以后面打算给init启动脚本和uci文件适配上支持环境变量传递参数值。只是没有调研过，向天灵大佬请教一下，过度使用环境变量而非命令传递参数会不会影响路由器性能（因为这类设备性能往往不是特别好）

1715173329 replied:
> 过度使用环境变量而非命令传递参数会不会影响路由器性能

Why?

v2raya 就是一个典型的利用环境变量进行配置的例子。

1715173329 replied:
> 目前这个代码会在每次启动时自动清除掉上次运行保存的log（即不是追加模式，是重写模式），虽然不会在stop_service时清除，但是也能达到差不多的效果，感觉可以不需要额外的回收机制

预期是在用户手动停止（stop）的时候回收这些内容。

AutoCONFIG replied:
已经加入stop_service便于回收全部日志。
此外
procd_add_jail hbbr log procfs sysfs
procd_add_jail_mount "/etc/ssl/"
procd_add_jail_mount_rw "/etc/rustdesk-server/"
procd_add_jail_mount_rw "/var/run/rustdesk-server/"
这个经过测试，没有发现什么问题。程序工作一切正常

AutoCONFIG committed:
rustdesk-server: remove shell wrapper for init
- Hardcode the log file path to /var/run/rustdesk-server
- Hardcode the key generation path to /etc/rustdesk-server
- Hardcode the database file path to /var/run/rustdesk-server
- Hardcode the configuration file path to /var/run/rustdesk-server
- Add fallback mechanism for MAC address cannot be obtained

Signed-off-by: Yun Wang <maoerpet@foxmail.com>
